batch/batch/worker/worker.py

```diff
@@ -2302,7 +2321,7 @@ def get_container_log_path(self, container_name: str) -> str:
         return self.log_file
 
     async def get_resource_usage(self) -> Dict[str, bytes]:
-        return {'main': ResourceUsageMonitor.no_data()}
+        return {'main': await read_resource_usage(self.worker.fs, self.resource_usage_file)}
```

Comment:
Hmm. I like what you did in the Bash job side where we only read this through the resource usage monitor. It does seem like there's no elegant way to achieve that here too.